.github/workflows/build-test.yml

```diff
@@ -129,3 +129,32 @@ jobs:
                 CLOUD_PROVIDER: ${{ matrix.cloud }}
                 TARGET_DOCKER_TEST_IMAGE: ${{ matrix.image }}
               run: ./ci/test.sh
+    test-ubuntu:
+      needs: build
+      name: Tests on Ubuntu
+      runs-on: ubuntu-latest
+      strategy:
+        fail-fast: false
+        matrix:
+          cloud: [ 'AWS', 'AZURE', 'GCP' ]
+          nodeVersion: ['18.x', '20.x']
+      steps:
+        - uses: actions/checkout@v1
+        - name: Download Build Artifacts
+          uses: actions/download-artifact@v1
+          with:
+            name: artifacts
+        - name: Use Node.js
+          uses: actions/setup-node@v3
+          with:
+            node-version: ${{ matrix.node-version }}
+        - name: Install dependencies
```

Comment:
shouldn't installing dependencies be a part of the test script?

Reply:
Script (test_ubuntu.sh) responsibility is testing. We have building as separated step.

Reply:
For mac we are not doing it. Additionally I see that test_component.sh install the dependencies so this step seems to be unnecessary

Reply:
ok